### PR TITLE
Loadsa pruning

### PR DESCRIPTION
**Pull request by Heegu-sama**
I removed many useless words/sentences/paragraphs. Also miscellaneous work such as spell checking, Capitalizing game terms, fiddling with page layouts.

This has created new gaps/empty space that should probably be filled with more graphics. I had to remove some of them out of the way previously and now during this. For instance the big vertical mummy isn't anywhere right now.

Combat's citadel section is currently missing its notice box about attacking walls because I couldn't think of where to place it back.

### CODE REVIEW

**Comment by Heegu-sama** [comment]
@qwrtln do you have time to re-fill some of the new gaps with art? I can do it myself tomorrow at the earliest.

**Comment by qwrtln** [comment]
> @qwrtln do you have time to re-fill some of the new gaps with art? I can do it myself tomorrow at the earliest.
Please do it tomorrow, I'll help you with issues.

File: sections/units.tex [hunk 1]
```diff
@@ -65,11 +52,8 @@ \subsection*{Unit Card Anatomy}
 \end{scaledfigure}
 
 \begin{center}
-  \includegraphics[width=\linewidth]{\cards/units.png}
+    \includegraphics[width=\linewidth]{\cards/units.png}
```
Comment (qwrtln) [inline sections/units.tex:55]:
```suggestion
  \includegraphics[width=\linewidth]{\cards/units.png}
```

**Review by qwrtln — APPROVED** [review]
Fake, one of the pictures does not represent a unit available in the game 🤪

**Comment by qwrtln** [comment]
Hold on, printable build is messed up now... I'll get back to it tomorrow.

image

**Review by qwrtln — COMMENTED** [review]
Need to fix those autoscaling images

**Review by qwrtln — CHANGES_REQUESTED** [review]
Right, miss click

**Review by qwrtln — APPROVED** [review]
All fixed

**Pull request merged**